Documentation/ceph-cluster-crd.md

```diff
@@ -121,6 +121,12 @@ The following storage selection settings are specific to Ceph and do not apply t
 - `databaseSizeMB`:  The size in MB of a bluestore database. Include quotes around the size.
 - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL). Include quotes around the size.
 - `journalSizeMB`:  The size in MB of a filestore journal. Include quotes around the size.
+- `osdsPerDevice`**: The number of OSDs to create on each device. High performance devices such as NVMe can handle running multiple OSDs. If desired, this can be overridden for each node and each device.
```

Comment:
The `**` seem out of place here.

Reply:
As we wrote on Slack, I'll "fix" this mark in one of my doc PRs.

Reply:
per our slack discussion, it's intended to be like a footnote marker for the note below.